src/main/java/agh/ics/oop/GrassField.java

```diff
@@ -15,6 +15,8 @@
 
 public class GrassField extends AbstractWorldMap implements IWorldMap {
     private int numberOfGrassFields;
+    private Vector2d lowerLeftOfGrass = new Vector2d(Integer.MAX_VALUE, Integer.MAX_VALUE);
+    private Vector2d upperRightOfGrass = new Vector2d(Integer.MIN_VALUE, Integer.MIN_VALUE);
```

Comment:
ponownie nie ma potrzeby, żeby to były atrybuty - wystarczy przecież żeby `MapBoundary` się tym zajmowało.